test/services/HolderOfAuthorisationCodeSpec.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import uk.gov.hmrc.play.test.UnitSpec
+
+class HolderOfAuthorisationCodeSpec extends UnitSpec {
+
+  "Holder of Authentication Code" should {
```

Comment:
So here we can change too, from `Holder of Authentication` to `Holder of Authorisation`

Reply:
Fixed